esm-loader.mjs

```diff
@@ -6,9 +6,16 @@
 import newrelic from './index.js'
 import shimmer from './lib/shimmer.js'
 import loggingModule from './lib/logger.js'
+import NAMES from './lib/metrics/names.js'
 
 const logger = loggingModule.child({ component: 'esm-loader' })
 
+if (newrelic.agent) {
+  newrelic.agent.metrics
+    .getOrCreateMetric(`${NAMES.SUPPORTABILITY.FEATURES}/ESModuleLoader`)
```

Comment:
I think we typically like to abstract metric names into that lib/metric/names.js file.  Grepping our agent code this is true for everything but 

```
sampler.js
166:          .getOrCreateMetric(NAMES.SUPPORTABILITY.DEPENDENCIES + '/NoNativeMetricsModule')
```